README.md

```diff
@@ -1,2 +1,129 @@
-# BookStore-React
-A BookStore project with react.
+# Portfolio<a name="readme-top"></a>
+
+<div align="center">
+  <h1> Book Store </h1>
+  <br/>
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
```

Comment:
- [ ] Kindly delete this link to the `Live Demo` section since we have deleted it, this link goes no where